Dockerfile

```diff
@@ -1,12 +1,18 @@
-FROM golang:1.5
+FROM golang:1.8
 RUN apt-get update && apt-get install -yqq aspell aspell-en libaspell-dev tesseract-ocr tesseract-ocr-eng imagemagick optipng exiftool libjpeg-progs webp
 ADD docker/meme.traineddata /usr/share/tesseract-ocr/tessdata/meme.traineddata
 ADD docker/imagemagick_policy.xml /etc/ImageMagick-6/policy.xml
 RUN mkdir -p /etc/mandible /tmp/imagestore
 ENV MANDIBLE_CONF /etc/mandible/conf.json
-ENV GO15VENDOREXPERIMENT 1
 ADD . /go/src/github.com/Imgur/mandible
 WORKDIR /go/src/github.com/Imgur/mandible
+
+RUN wget "https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-cpu-linux-x86_64-1.0.0.tar.gz"
```

Comment:
Yeah, good call! It will run on GPU support if run outside of docker on that AMI. I also added a `Dockerfile.gpu` that includes the CUDA libraries, it needs to be run with [nvidia-docker](https://github.com/NVIDIA/nvidia-docker) to allow it access to the host's GPUs. 

I thought it was a good idea to have a separate Dockerfile for cpu and gpu support since the gpu container does a lot of extra work. Let me know if you think they should be merged!